test/extended/authorization/scc.go

```diff
@@ -30,7 +30,12 @@ var _ = g.Describe("[sig-auth][Feature:SCC][Early]", func() {
 		suiteStartTime := exutil.SuiteStartTime()
 		var preTestDenialStrings, duringTestDenialStrings []string
 
+		eventsAfterTime := exutil.LimitTestsToStartTime()
```

Comment:
Reading the code around, it has been modified since I first authored it.  What is the difference between this and `preTestDenialStrings` and `duringTestDenialStrings`?

Reply:
Looking at failure https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25774/pull-ci-openshift-origin-master-e2e-aws-disruptive/1387235806111862784 , any chance they simply forgot to create their rolebinding?  Maybe they aren't reactive?  Perhaps checking creation time of the rolebinding against the events?

At the point when this test is running, is the cluster supposedly stable?

Reply:
> Reading the code around, it has been modified since I first authored it. What is the difference between this and `preTestDenialStrings` and `duringTestDenialStrings`?

Is the time on this bugged perhaps?  Maybe try logging that time first so you can see when the failures happened versus the suiteStartTime?

Reply:
Arg, this test is already checking the start time. This PR is not needed, the failure is legit indicating an issue with post-disruptive scc propagation.

Reply:
And now I see that the test is checking `TEST_SUITE_START_TIME`, and this PR is adding a check for `TEST_LIMIT_START_TIME`.

Reply:
SUITE_START_TIME is set by openshift-tests.  LIMIT_START_TIME is set in release only.

Reply:
synthetics should infer "start time" from the events they are passed.

Reply:
@smarterclayton I'm not sure how to act on your comments. Is this PR unnecessary, or is more work involved?